### DISCUSSION TITLE
Welcome to MiniStar-Nano Discussions!

### DISCUSSION THREAD

**Discussion by magicjellybeanfpga** [magicjellybeanfpga/MiniStar-Nano#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jgraham1989** [comment]
I'm working on a robotic project using nvidia jetson nano dev kit, nvidia jetson tx2 dev kit, a few raspberry pi picos and a ton of modules and sensors and I bought this on a whim. No idea what it does exactly. I thought it was a bridge or a uart device originally that I could use to link the tx2 with jetson nano.

**Reply by jgraham1989** [reply]
Still can't find out how to use it or what it does. Waste

**Reply by magicjellybeanfpga** [reply]
Hi Jacob,

I see that you purchased the MiniStar-Nano but are not sure what you can do with it. 

You will most likely need the experiment carrier to do any of the available demos. You can check out some demo videos here:

https://magicjellybeanfpga.github.io/mjb/demos.html

**Comment by jgraham1989** [comment]
Hey thanks for the response!
So I have purchased the tx2 dev kit jetson nano dev kit along with
countless sensors, cameras, a Yahboom nano jetbot , arduino raspberry pi
pico ws. Etc. are you curious as to what I’m building? I was hoping I could
incorporate something small but affective to do one of three things: store
the store the sdk manager on it so if something goes wrong with power alone
it can boot the embedded machines back to their operating state or a device
that can act as a probability indicator and attach a value marker as data
flows through it ( so it will need to analyze variables quickly , and or
lastly a device that can act as a bridge between the tx2 and nano
converting data both ways.

Iv just no pulled out the mini star and started to scketch it to attempt to
crack the code of what it is myself.

**Comment by jgraham1989** [comment]
I gues my question is can this be used with the nvidia dev kits mentioned?
Or anything mentions in my response ?
